### PR TITLE
ak36: vlans and services

### PR DESCRIPTION
**Pull request by pktpls**

### CODE REVIEW

**Review by FFHener — APPROVED** [review]
Lgtm, but would still like to wait for a second review

**Comment by pmelange** [comment]
The generated config looks good, except that interface mesh_bbbvpn is also in the babled config.  It doesn't need to be there at all as the bbbvpn doesn't speak babel. Commit https://github.com/freifunk-berlin/bbb-configs/commit/db1af477f31090016ef6af0bf0f6d4a9a7893c53 should prevent the babeld config section from being created.

File: locations/ak36.yml
Comment (Noki) [inline locations/ak36.yml:116]: please make IPv6 a comment as we've done with https://github.com/freifunk-berlin/bbb-configs/blob/master/locations/strom.yml#L82 so no babeld config will be generated for this interface

**Pull request merged**
